src/python/pants/engine/mapper.py

```diff
@@ -192,3 +192,7 @@ def __repr__(self):
   @memoized_property
   def exclude_patterns(self):
     return tuple(re.compile(pattern) for pattern in self.exclude_target_regexps)
+
+  def __str__(self):
+    parser_name = self.parser.__class__.__name__
+    return f'AddressMapper(parser of type {parser_name}, ...)'
```

Comment:
Is this `...` intentional?

Reply:
Yes, although it doesn't appear to be getting used at all to actually display the node in the visualized graph, so it's not clear that this is needed.

Reply:
The original rationale for this was that the default `__str__` for this object was massive and was making its node take up a huge amount of space in the output pdf. If the `__str__` is no longer being used here, we may be ok killing it.